model/src/main/proto/subtitled_unicode.proto

```diff
@@ -0,0 +1,11 @@
+syntax = "proto3";
+
+package model;
+
+option java_package = "org.oppia.app.model";
+option java_multiple_files = true;
+
+message SubtitledUnicode {
```

Comment:
Please add doc comments for the message & fields.

Reply:
done